pkg/payload/v1alpha1/delete.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright © 2018-2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/container-storage-interface/spec/lib/go/csi"
+)
+
+// DeleteVolumeResponseBuilder helps building an
+// instance of csi DeleteVolumeResponse
+type DeleteVolumeResponseBuilder struct {
```

Comment:
If we are never going to build a response instance, then a dedicated builder is not required

Reply:
we are builidng response for both create and delete.

Reply:
we are building the response instance while replying back.